apps/dgst.c

```diff
@@ -544,14 +569,62 @@ static const char *newline_escape_filename(const char *file, int *backslash)
     return (const char*)file_cpy;
 }
 
+static void print_out(BIO *out, unsigned char *buf, size_t len,
+                      int sep, int binout,
+                      const char *sig_name, const char *md_name, const char *file)
+{
+    int i, backslash = 0;
```

Comment:
```suggestion
    int backslash = 0;
    size_t i;
```
and avoid casting in the for loops?  It doesn't seem to be used anywhere else.

Reply:
I really think this one should get done...

Reply:
done